frame/vesting/src/tests.rs

```diff
@@ -1155,3 +1155,39 @@ fn vested_transfer_less_than_existential_deposit_fails() {
 		);
 	});
 }
+
+#[test]
+fn remove_vesting_schedule() {
+	ExtBuilder::default().existential_deposit(ED).build().execute_with(|| {
+		let user3_free_balance = Balances::free_balance(&3);
+		let user4_free_balance = Balances::free_balance(&4);
+		assert_eq!(user3_free_balance, 256 * 30);
+		assert_eq!(user4_free_balance, 256 * 40);
+		// Account 4 should not have any vesting yet.
+		assert_eq!(Vesting::vesting(&4), None);
+		// Make the schedule for the new transfer.
+		let new_vesting_schedule = VestingInfo::new(
+			256 * 5,
+			64, // Vesting over 20 blocks
```

Comment:
20? 64?

Reply:
The ED is set to 256. I think its more understandable if you use `ED` instead of 256 and write `(ED * 5) / 20`.

Reply:
@ggwpez done, thanks